Chamasma/DesignSystem/Sources/DesignSystem/Components/ComponentViews/Buttons/RoundButton/RoundButtonModel.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  RoundButtonModel.swift
+//
+//
+//  Created by Mariam Ormotsadze on 05.05.23.
+//
+
+import Common
+import SwiftUI
+
+struct RoundButtonModel {
+    let image: RoundButtonImage
+    let state: RoundButtonState
+    let size: RoundButtonSize
+
+    init(
+        image: RoundButtonImage,
+        state: RoundButtonState,
+        size: RoundButtonSize
+    ) {
+        self.image = image
+        self.state = state
+        self.size = size
+    }
+}
+
+extension RoundButtonModel {
+    enum RoundButtonImage {
+        case plus
+        case search
+        case backArrow
+        case close
+
+        var name: Image {
```

Comment:
Name should not return Image

Reply:
can you, please, suggest better name? I have been thinking and was not able to figure it out